### PR TITLE
Use new JS Solr Utils library

### PR DESCRIPTION
**Pull request by rderik**
The code has been updated to use the new JS Solr Utils library that
reduces the code and makes the partial be cleaner.

This commit contains the following changes:

Refactors code for fancyTree and collapsible plugin
The name of the plugins was changed on kmaps_engine and now the change
to those names has been reflected in this commit.
fancyTree was renamed to kmapsRelationsTree
collapsibleList was renamed to kmapsCollapsibleList

Update code to on related for summary
Cleaned code for new update on solr-utils library

### CODE REVIEW

File: app/views/features/_related.html.erb [hunk 1]
```diff
@@ -70,127 +38,51 @@
...
+      var feature_label = $('<%= feature_label %>')[0].innerText;
+      var feature_path = "<%= features_path %>/%%ID%%";
+      relatedSolrUtils.addSubjectsSummaryItems(feature_label,feature_path,'parent',result);
+      relatedSolrUtils.addSubjectsSummaryItems(feature_label,feature_path,'child',result);
+      //relatedSolrUtils.addSubjectsSummaryItems(feature_label,feature_path,'other',result);
+
       // Functionality for columnizer
       // dontsplit = don't break these headers
-      $('.subjects-in-subjects').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
+      $('.places-in-places').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
```
Comment (amontano) [inline app/views/features/_related.html.erb]: I think the previous line "subjects-in-subjects" was correct and not places-in-places given that we are in subjects_engine!

Comment (amontano) [reply app/views/features/_related.html.erb]: Others line below also referencing places-in-places. Check whole file.

File: app/views/features/_related.html.erb [hunk 2]
```diff
@@ -248,40 +139,5 @@
...
-  top:0;
-  right:0;
-  background-image: none;
-}
-li.collapsible_list_header::before{
-  display: none !important;
-}
-</style>
-<%= javascript_include_tag('collapsible_list/jquery.collapsibleList') %>
+<%= stylesheet_link_tag('subjects_engine/related') %>
```
Comment (amontano) [inline app/views/features/_related.html.erb]: better to have stylesheet as above as possible in this file.

File: app/views/features/_related.html.erb (hunk 1 above)
Comment (rderik) [inline app/views/features/_related.html.erb]: Change this selector to be subjects-in-subjects,

File: app/views/features/_related.html.erb [hunk 3]
```diff
@@ -70,127 +38,51 @@
...
-      $('.subjects-in-subjects').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
+      $('.places-in-places').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
       // dontend = don't end column with headers
-      $('.subjects-in-subject').find('.column > h6, .column > ul > li').addClass("dontend");
-      $('.subjects-in-subjects').find('.feature-block').addClass("dontsplit");
-      $('.kmaps-list-columns:not(.places-in-subjects):not(.already_columnized)').addClass('already_columnized').columnize({
-        width: 330,
-        lastNeverTallest : true,
-        buildOnce: true
+      $('.places-in-places').find('.column > h6, .column > ul > li').addClass("dontend");
```
Comment (rderik) [inline app/views/features/_related.html.erb]: All this sleecteros form line 76-82 should use subjects not places

File: app/views/features/_related.html.erb (hunk 2 above)
Comment (rderik) [inline app/views/features/_related.html.erb]: Move CSS tag to the top of the partial

**Review by rderik — COMMENTED** [review]
Make the small changes detailed on the comments before the merge

**Review by amontano — APPROVED** [review]

**Pull request merged**